generated/thrift/rpc.thrift

```diff
@@ -133,12 +133,12 @@ struct Segment {
 }
 
 struct FetchTaggedRequest {
-	1: required IdxQuery query
-	2: required i64 rangeStart
-	3: required i64 rangeEnd
-	4: required bool fetchData
-	5: optional i64 limit
-	6: optional TimeType rangeTimeType = TimeType.UNIX_SECONDS
+	1: required string nameSpace
```

Comment:
Make this `binary` instead of `string` considering `IdxQuery` and it composite child type `IdxTagFilter` is using binary for `tagName`, etc.

Reply:
sure thing

Reply:
done